### DISCUSSION TITLE
AI發展與人權議題

### DISCUSSION THREAD

**Discussion by xipeiguo** [1101-GS4719-C/Python-Community-GS4719-C#2040] [General]
https://www.ettoday.net/news/20240807/2793048.htm
文章中提及阿根廷政府欲使用AI預測未來犯罪，我想我們不可否認AI確實可以帶給人們生活中更多的便利性，但隨之而來的人權議題，更是我們不可忽視的一大問題，網路上監控的言論自由，抑或是生活中的人身自由等，如何在人權與科技發展和應用中取得平衡，是政府需嚴謹應對的一大難題，而我非常期待AI未來給予我們的生活便利，然而前提是政府亦須先設立充足應對各種情形的法律!

**Comment by ds0w0** [comment]
我認同你的想法，當一個新技術出現時勢必也會有挑戰，網路普及時也帶來不小的資訊安全與道德問題，雖然仍是現在進行式，但我覺得隨著越來越多人的關注，我們會找到便利與權益的平衡。不過前提是沒有濫權或猖獗的非法行為，畢竟現在通訊發達，個資一不小心就洩漏出去了。

**Comment by william-60760** [comment]
我也同意政府應做好法律準備的看法。除此之外，我覺得利用 AI 預測犯罪的準確度也是值得探討的問題。在犯罪手法推陳出新的現代下，我認為歷史犯罪數據不一定足夠充足使AI 準確判斷潛在罪犯，而且還有相關不等於因果的因素需考量。退一步來講，即使 AI 能 100% 預測潛在罪犯，警方也不應直接進行抓捕。畢竟在AI 預測的「未來犯罪」尚未發生或預謀下，人們仍然是清白無罪的。

**Reply by ds0w0** [reply]
畢竟就算是AI推測的結果，也不是最真實的情況，如同證人的說詞無法定罪，我們仍然要落實無罪推定原則。

**Comment by MingWei174** [comment]
我也期許AI未來能夠不斷發展為人類帶來許多生活上的便利，但使用AI技術預測犯罪也無法保證其預測結果的準確性，因此當中產生的網路安全以及人權問題是需要政府事先規劃應對各種突發情況的法律和處理手段來將負面效果降至最低。

**Comment by hunteryang0108** [comment]
我認為只要是預測，都不可能取得百分百的可信度。現實生活中的事件是會因為很多因素而改變的，就算AI能夠預測事件的發生機率，也不代表它一定會發生。因此各國當權者應該審慎思考如何利用AI，而不是如何依賴AI。

**Comment by CtyceT** [comment]
若是有法律的限制和保證之下控制AI對於人類的隱私的侵害的前提之下使AI發展，那麼AI就是一個對人類非常強大的輔助型工具。反之，若AI無法被制約，或是任其隨意發展，那麼這無疑是飲鴆止渴。

**Comment by Jerryguo888** [comment]
AI確實能帶來許多便利，但同時也伴隨著人權問題，像是網路監控和個人隱私。而科技的進步不應忽略法律和倫理的保障，政府需要制定相關法律，設立透明的監管機制，以確保科技應用不會侵害基本人權。同時，公眾也可以參與監督，以促進科技與人權的和諧共存。
